apps/workspaces/apis/export_settings/helpers.py

```
@@ -0,0 +1,139 @@
+from typing import List
+from django.db import transaction
+
+from apps.fyle.models import ExpenseGroup
+from apps.tasks.models import Error, TaskLog
+from apps.workspaces.models import Configuration
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def clear_workspace_errors_on_export_type_change(
```

Pylint (3.3.7)

[refactor] Too many branches (17/12)

(R0912)